src/gps/README.txt

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
This file seems like it has Markdown formatting, so should this be a .md file instead of .txt?

Reply:
Yeah, I'll change that.